hintparser.y

```diff
@@ -78,7 +78,7 @@ import (
 
 	/* TiDB hint names */
 	hintAggToCop              "AGG_TO_COP"
-	hintEnablePlanCache       "ENABLE_PLAN_CACHE"
+	hintDisablePlanCache      "DISABLE_PLAN_CACHE"
```

Comment:
How about `ignore_plan_cache` or `ignore_cached_plan`? `disable_plan_cache` sounds like a hint that can disable the plan cache, clear all the items in the cache.

@eurekaka How do you think?

Reply:
`ignore_plan_cache` sounds reasonable.

Reply:
rename done